mbed/src/iot_network.cpp

```diff
@@ -1,3 +1,8 @@
+/* mbed Microcontroller Library
```

Comment:
`mbed Microcontroller Library`

I would just remove this first line as it is just as it used to be when mbed was a library (mbed.lib).

Reply:
Ah good to know. This line appears in many of our libraries and examples...

Reply:
Should it look like
```c
/* Copyright (c) 2020 Arm Limited
 * SPDX-License-Identifier: Apache-2.0
 */
```
? Or
```c
/*
 * Copyright (c) 2020 Arm Limited
 * SPDX-License-Identifier: Apache-2.0
 */
```
?

Reply:
@0xc0170 Shall I use the second one which looks a bit better?

Reply:
Done